vignettes/articles/widget.Rmd

```diff
@@ -9,9 +9,9 @@ knitr::opts_chunk$set(
 )
 ```
 
-The function `leaflet()` returns a Leaflet map widget, which stores a list of objects that can be modified or updated later. Most functions in this package have an argument `map` as their first argument, which makes it easy to use the pipe operator `%>%` in the **magrittr** package, as you have seen from the example in the [Introduction](./).
+The function `leaflet()` returns a Leaflet map widget, which stores a list of objects that can be modified or updated later. Most functions in this package have an argument `map` as their first argument, which makes it easy to use the pipe operator `%>%` in the **magrittr** package, as you have seen from the example in `vignette("leaflet")`.
```

Comment:
Still keep the link to the article...

```suggestion
The function `leaflet()` returns a Leaflet map widget, which stores a list of objects that can be modified or updated later. Most functions in this package have an argument `map` as their first argument, which makes it easy to use the pipe operator `%>%` in the **magrittr** package, as you have seen from the example in [`vignette("leaflet")`](./).
```

Reply:
`vignette()` autolinks on pkgdown ;)  https://pkgdown.r-lib.org/articles/pkgdown.html#metadata See the link there to `vignette("customize")`  
![image](https://github.com/rstudio/leaflet/assets/52606734/22c38c4e-4be8-4a8e-ae50-b98b9dda469e)

Reply:
Thank you for another TIL!

**Resolving without accepting suggestion